swim/node_bootstrap_test.go

```diff
@@ -74,21 +74,20 @@ func (s *BootstrapTestSuite) TestBootstrapJoinsTimeOut() {
 }
 
 func (s *BootstrapTestSuite) TestBootstrapDestroy() {
-	var lerr lError
+	errChan := make(chan error)
 
 	go func() {
 		_, err := s.node.Bootstrap(&BootstrapOptions{
 			Hosts:       fakeHostPorts(1, 1, 1, 10),
 			JoinTimeout: time.Millisecond,
 		})
-		lerr.Set(err)
+		errChan <- err
 	}()
 
-	time.Sleep(2 * time.Millisecond)
 	s.node.Destroy()
```

Comment:
I think if Destroy should go first, we would benefit from moving it up. It makes it easier to understand what we're testing for.